Nitrox.Launcher/Platforms/MacOS/Info.plist

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>CFBundleIconFile</key>
+        <string>Nitrox.icns</string>
+        <key>CFBundleIdentifier</key>
+        <string>com.nitrox.launcher</string>
+        <key>CFBundleName</key>
+        <string>Nitrox</string>
+        <key>CFBundleDisplayName</key>
+        <string>Nitrox</string>
+        <key>CFBundleVersion</key>
+        <string>1.8.0</string>
+        <key>LSMinimumSystemVersion</key>
+        <string>12.0</string>
+        <key>CFBundleExecutable</key>
+        <string>Nitrox.Launcher</string>
+        <key>CFBundleInfoDictionaryVersion</key>
+        <string>6.0</string>
+        <key>CFBundlePackageType</key>
+        <string>APPL</string>
+        <key>CFBundleShortVersionString</key>
+        <string>1.8</string>
+        <key>NSPrincipalClass</key>
+        <string>NSApplication</string>
+        <key>NSHighResolutionCapable</key>
+        <true/>
+    </dict>
+</plist>
```

Comment:
Would be good to generate this from within msbuild (and embed it at compile target?). Then the info can always sync with msbuild properties.

(Though I'm unfamiliar with how plist is used by OSX and why it's in `/Platforms/MacOS/`.)

Reply:
Good idea, could definitely be generated by hand, that's what Xamarin, MAUI, ... are doing under the hood.

Info.Plist it's just a manifest in apple ecosystem that will tell information about the application, Name, Version, Mininum required OS version, Executable Name, Executable type.
Entitlement will just ask for specific permissions (android like), those are mainly used when apple need to notarize one of your app during review session

Reply:
Will reconsider adding manifest generation once this once is done